libraries/lib-datahandler/src/main/java/com/siemens/sw360/datahandler/businessrules/ReleaseClearingStateSummaryComputer.java

```diff
@@ -50,7 +50,10 @@ public static ReleaseClearingStateSummary computeReleaseClearingStateSummary(Lis
     }
 
     static Void addReleaseWithStates(ReleaseClearingStateSummary summary, ViewedState globalState, ViewedState myTeamState, ViewedState otherTeamState) {
-        if (globalState.present()) { // if the release has its clearing state field set use it and ignore the rest
+        // if the release has its clearing state field set to REPORT_AVAILABLE or APPROVED use it and ignore the rest
+        // otherwise, look at what fossology has to say
+        // if all else fails, default to NEW
+        if (globalState.present() && globalState.compareTo(ViewedState.REPORT_AVAILABLE) >= 0) {
```

Comment:
This is exactly what `Optional.filter` does:

>**Parameters:**
predicate - a predicate to apply to the value, if present
**Returns:**
an Optional describing the value of this Optional if a value is present and the value matches the given predicate, otherwise an empty Optional

this could make the code shorter.